V1.4 Check Command Update

pinjuen opened the pull request:
- took out check in time and working rate from Person Card
- different colors for check in and out status
- took out NRIC and Password from command
- not allow more than one repeated prefix
- updated user guide and developer guide accordingly.

pinjuen committed:
update

pinjuen committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-T16-3/main into uiUpdate

# Conflicts:
#	src/main/java/seedu/address/model/util/SampleDataUtil.java
#	src/test/java/seedu/address/testutil/TypicalPersons.java

pinjuen committed:
updated screenshots in user guide accordingly

pinjuen committed:
update on test

speezy37 reviewed (dismissed):
Awaiting for UI changes

pinjuen committed:
updates on person card:
- removed working rate and check in time on person card
- changing colour when checking in/out

pinjuen committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-T16-3/main into uiUpdate

pinjuen committed:
updated user guide screenshots accordingly

pinjuen committed:
Merge branch 'master' into uiUpdate

pinjuen committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-T16-3/main into uiUpdate

# Conflicts:
#	docs/UserGuide.adoc
#	src/main/java/seedu/address/model/util/SampleDataUtil.java

pinjuen committed:
Merge branch 'uiUpdate' of https://github.com/pinjuen/main into uiUpdate

# Conflicts:
#	docs/UserGuide.adoc

pinjuen committed:
- took out NRIC and Password
- not allow more than one prefix
- updated user guide

pinjuen committed:
- updated developer guide
- fix checkstyle

pinjuen committed:
update developer guide

pinjuen committed:
fix appveyor

jinyao-lee reviewed (approved):
UI looks good to me.

pinjuen committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-T16-3/main into uiUpdate
